sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/KeyVaultAdministrationModelFactory.cs

```diff
@@ -52,6 +52,16 @@ public static KeyVaultRoleAssignmentProperties KeyVaultRoleAssignmentProperties(
             return new KeyVaultRoleAssignmentProperties(scope, roleDefinitionId, principalId);
         }
 
+        /// <summary> Initializes a new instance of KeyVaultSetting. </summary>
+        /// <param name="name"> The account setting to be updated. </param>
+        /// <param name="content"> The value of the pool setting. </param>
+        /// <param name="settingType"> The type specifier of the value. </param>
+        /// <returns> A new <see cref="Administration.KeyVaultSetting"/> instance for mocking. </returns>
+        public static KeyVaultSetting KeyVaultSetting(string name = null, string content = null, KeyVaultSettingType? settingType = null)
```

Comment:
Is `content` always a `string`?

Reply:
Yes. Their swagger defines `value` as a string, but I renamed it to `content` to match FR and because it couldn't be `Value` if that was returning our `KeyVaultSettingValue` type.